samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -25,6 +28,7 @@ BUILD_ASSERT(IS_ENABLED(CONFIG_SCAN_SELF) || IS_ENABLED(CONFIG_SCAN_OFFLOAD),
 #define INVALID_BROADCAST_ID      (BT_AUDIO_BROADCAST_ID_MAX + 1)
 #define SYNC_RETRY_COUNT          6 /* similar to retries for connections */
 #define PA_SYNC_SKIP              5
+#define NAME_LEN 30
```

Comment:
Since we want to compare with `CONFIG_BT_TARGET_BROADCAST_NAME`, could we use the length of `CONFIG_BT_TARGET_BROADCAST_NAME` instead? 

Or at least `NAME_LEN` should be `>= sizeof(CONFIG_BT_TARGET_BROADCAST_NAME)`, right? 

So we could perhaps set `NAME_LEN = sizeof(CONFIG_BT_TARGET_BROADCAST_NAME) + 1`, so that we ensure that we can always compare the two strings.